src/Config.ts

```diff
@@ -0,0 +1,29 @@
+import convict from 'convict'
+
+const config = convict({
+  env: {
+    doc: 'The application environment.',
+    format: ['production', 'sandbox', 'development'],
+    default: 'sandbox',
+    env: 'NODE_ENV',
+  },
+  settings: {
+    merchantId: {
```

Comment:
and this would come from admin settings.   Maybe ApiBasePath , and some other made up thing